### ISSUE TITLE
About the validation set

### ISSUE THREAD

**Issue by zhuye98** [PLAN-Lab/CheXRelFormer#2]
Hi Amarachi,

Your work on disease progression detection are very interesting and inspiring!

I'm curious why you didn't use the Gold Standard Dataset for evaluation (Although I know the Gold Standard Dataset is mainly for evaluating the performance of the auto-labelers...). The auto-extracted progression labels in the silver dataset seem quite noisy according to Table 4 in the paper "Chest ImaGenome Dataset for Clinical Reasoning." Just a thought, eager to hear your response. Thank you!

**Comment by AmarachiBlessingMbakwe** [comment]
Hi Zhu,

Thank you for your kind words and interest in our disease progression detection work.
We used the official training, validation, and test split in the MIMIC-CXR dataset to ensure alignment with existing benchmarks or baselines.

It would be interesting to study the performance on the Gold Standard Dataset.
 
Thank you again for your thoughtful question.

**Comment by zhuye98** [comment]
Hi Amarachi,

Thank you for your kind response! I have no more questions at the moment~
